Lib/ufo2ft/__init__.py

```diff
@@ -5,8 +5,8 @@
 from fontTools.misc.py23 import *
 
 from ufo2ft.featureCompiler import FeatureCompiler
-from ufo2ft.kernFeatureWriter import KernFeatureWriter
-from ufo2ft.markFeatureWriter import MarkFeatureWriter
+from ufo2ft.featureWriter.kernFeatureWriter import KernFeatureWriter
+from ufo2ft.featureWriter.markFeatureWriter import MarkFeatureWriter
```

Comment:
from ufo2ft.featureWriter (writers?) import Kern*, Mark* would be nice (reexport)

Reply:
Done.